### ISSUE TITLE
"BACKUP_PATH" in config.yml is not respected

### ISSUE THREAD

**Issue by yue-dongchen** [LuteOrg/lute-v3#431]
**Description**

I set up cloud synchronisaton of the data folder between my Windows and macOS machines. In the config.yml of each installation, "DATAPATH" is respected, but "BACKUP_PATH" is not. The directory is instead stored in the database, which necessitates me to disable backups under this setup.

**Extra software info, if not already included in the Description:**
Lute 3.4.0 without Docker

**Comment by jzohrab** [comment]
Hi @yue-dongchen , thx for the issue.

The BACKUP_PATH was intended to only be used for Docker, but the config file comments aren't clear at all.

Can you clarify what it is that you're doing?  You share the db between two different computers, and expect the BACKUP_PATH to be used, regardless of the setting in the Settings page?

**Comment by jzohrab** [comment]
Yes, this is required for situations where someone uses dropbox to sync the db to different computers, because the backup paths will be different for different areas.

_however_, if a machine uses Docker, it should _not_ use the backup_path, since that needs to be the specific mounted folder in the container. :-P
